### ISSUE TITLE
Add Astra DB to list of driver configurations

### ISSUE THREAD

**Issue by melienherrera** [dbeaver/dbeaver#15006]
**Is your feature request related to a problem? Please describe.**
Wanted to add Astra DB as one of the options of driver configurations built into DBeaver.

**Describe the solution you'd like**
Below is the driver description for Astra DB. 

**DRIVER DESCRIPTION:**
- Driver Name: `Astra DB`
- Driver Type: `Generic`
- Class Name: `com.simba.cassandra.jdbc42.Driver`
  - Library:   [SimbaCassandraJDBC42-2.0.12.1013.zip](https://github.com/dbeaver/dbeaver/files/7770786/SimbaCassandraJDBC42-2.0.12.1013.zip)

**URL TEMPLATE:**

`jdbc:cassandra://;AuthMech=2;TunableConsistency=6;SecureConnectionBundlePath=/path/to/secure-connect-bundle.zip`
Note: User has to specify the exact path to their secure connect bundle. \

Goal is to be added on this screen:
![Screen Shot 2022-02-02 at 9 20 02 AM](https://user-images.githubusercontent.com/85907755/152204321-2198dfb3-1078-413f-83f3-b23b617423b4.png)

Note: Came from issue #14891

**Comment by tati-kru** [comment]
@melienherrera it seems that provided driver is not open-source. We cannot add this to DBeaver Community.

**Comment by pieterhumphrey** [comment]
@tati-kru Hello, we'd like to re-open this for your consideration.
We have a permissive license that allows redistribution, similar to the Elastic license that you already use.
Can we have the lawyers review our EULA please?  send me an email at pieter dot humphrey at datastax dot com please!  thank you.

**Comment by melienherrera** [comment]
@Destrolaric thank you for reopening this

**Comment by flambchap** [comment]
Hello @pieterhumphrey. I'm taking a look at https://www.datastax.com/legal/simba-odbcjdbc-driver-license-terms. Paragraph 2 (Restrictions) states: 

> Licensed Software is confidential and copyrighted. Except as expressly permitted under this Agreement, you agree not to assign, distribute, transfer...

It doesn't look permissive to me. I believe I'm looking at the wrong license terms. Could you please show me the right ones?

**Comment by pieterhumphrey** [comment]
@akilovich yes, I'm trying to find the right license, will link you to it - thank you!
